### PR TITLE
YogStation: Making evac more robust 

### PR DESCRIPTION
**Pull request by EIB0N**
# General Documentation

### Intent of your Pull Request

This is part 2 of the Yogstation changes this time I am doing the exit

### Why is this change good for the game?

Better map happier players

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Exit has been remodeled

### What should players be aware of when it comes to the changes your PR is implementing?

Have fun!

### What general grouping does this PR fall under? 

Evacuation Changes 

download

# Changelog

Exit has been redone!

:cl:  

tweak: changed the design of exit to look cooler 

/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
replace one of those cig vendors with the arcade machine thats supposed to be at evac

**Comment by Cartlord** [comment]
Looks too flashy to me, and I dislike the lack of chairs.

**Comment by Xoxeyos** [comment]
Tone down on the colors, it feels like I'm staring into a kaleidoscope. The pod bay doesn't need the blue and green, as the clerk's office already has that, the outside of the security door doesn't need that red outcropping.

TL;DR I suggest you cut back on the colors, or make colors that don't clash so harshly, within an enclosed environment.

**Comment by patpol4** [comment]
Alternative
https://github.com/yogstation13/Yogstation/pull/11473

**Comment by EIB0N** [comment]
So cut back on the seating arrangement now its just chairs. Added a small arcade area, cut back on the color by a good amount

**Comment by Cartlord** [comment]
I am pleased by the chairs, and by the arcade machine now being one of the only ones on the station that's rotated properly.

**Comment by EIB0N** [comment]
> I am pleased by the chairs, and by the arcade machine now being one of the only ones on the station that's rotated properly.

rotated properly for now ;)

**Comment by athena413** [comment]
i actually like how this removes the chapel exit to escape, it makes the hallway actually be a nice little point where shit can go wrong

just make sure that the chapel gets a front entrance or some other kind of entrance

**Comment by alexkar598** [comment]
Not a massive fan of how restrictive this on engineering tbfh. I have alot of fun remodelling escape as engineer and this kinda prevents all of it with 50 railings and most importantly te decals you cant remove

**Comment by Marmio64** [comment]
really cool looking but as alexkar said, departures is prime real estate for engineering projects and this would make them much harder to do and maybe even impossible.

also i actually like having the exit into departures for the chapel.
the PR is not bad by any means and the map looks good but its fun to build in departures and its good to not over rely on easily broken illusions

**Comment by EIB0N** [comment]
> really cool looking but as alexkar said, departures is prime real estate for engineering projects and this would make them much harder to do and maybe even impossible.
> 
> also i actually like having the exit into departures for the chapel.
> the PR is not bad by any means and the map looks good but its fun to build in departures and its good to not over rely on easily broken illusions


do you think I should just add the illusion to the turf itself so the only way it breaks is by breaking the floor? Also rp/gameplay wise is it reasonable to have engineering projects obstructing evac ; every server is different gameplay wise so let me know

**Comment by Marmio64** [comment]
> > really cool looking but as alexkar said, departures is prime real estate for engineering projects and this would make them much harder to do and maybe even impossible.
> > also i actually like having the exit into departures for the chapel.
> > the PR is not bad by any means and the map looks good but its fun to build in departures and its good to not over rely on easily broken illusions
> 
> do you think I should just add the illusion to the turf itself so the only way it breaks is by breaking the floor? Also rp/gameplay wise is it reasonable to have engineering projects obstructing evac ; every server is different gameplay wise so let me know

you could try
and yes it is reasonable for people to make engineering projects in evac so long as it doesn't block people from getting to to the shuttle.

**Comment by EIB0N** [comment]
just realized you guys actually use the empty space so ill just close the pr ty for the input guys
